extensions/legit/porcelain-fossil.lisp

```diff
@@ -0,0 +1,137 @@
+(uiop:define-package :lem/porcelain-fossil
```

Comment:
Going with one-package per file (git, fossil, hg) goes against the simplification of my latest commit (where I deleted the distinction between peek-legit and legit).

The "use" of lem/porcelain might speak against the need of another package.

Is there a hard reason? What do you want encapsulation for, parameters?

Reply:
> Going with one-package per file (git, fossil, hg) goes against the simplification of my latest commit (where I deleted the distinction between peek-legit and legit).

I mean yes, we do have the extra "complication" of having different git/hg/fossil package declarations, but to me having the implementations be properly isolated seems a more significant *simplification*.

And I don't think the `:use` of `lem/porcelain` indicates any package-squashing: `lem/porcelain`'s whole point now is to define an interface. It's only natural that implementors need to know about everything in it/

> Is there a hard reason? What do you want encapsulation for, parameters?

Hard reason? Well I guess not, other than it making a lot of sense: there's no reason for `git` code to have access to `run-fossil`, or for `hg` stuff to know of `*git-base-arglist*: you get the idea.

Reply:
Seems like we're done here, resolving